### ISSUE TITLE
Buttons not labeled correctly when using VoiceOver

### ISSUE THREAD

**Issue by aaron-gh** [AppHouseKitchen/AlDente-Charge-Limiter#1082]
**Describe the bug**
All the buttons in the application's popup window have the accessibility label "popover"
**To Reproduce**
1. Turn on the voiceOver screen reader, with command + f5
2. 2. open the application's popup window from the menubar
3. 3. Use the voiceover keys, control plus option, and the cursor keys left and right arrow to explore the window.
4. 4. All the buttons will be read as "popover"
**Expected behavior**
The button labels should describe the function of the button.
**Screenshots**
If applicable, add screenshots or screen recordings to help explain your problem.

**MacBook (please complete the following information):**
 - MacBook model (https://support.apple.com/en-us/HT201608)
 - macOS version (https://support.apple.com/en-us/HT201260)
 - AlDente Pro or AlDente Free?
 - AlDente Version?
 - List of all enabled/disabled settings in AlDente:
This doesn't really apply, applies to all moddels/versions
**Additional context**
